onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -151,9 +156,26 @@ def aten_cross_entropy_loss(
 
 
 def aten_elu(
-    self: TensorType, alpha: int = 1, scale: int = 1, input_scale: int = 1
+    self: FloatType,
+    alpha: float = 1.0,
+    scale: Annotated[float, Is[lambda x: x == 1.0]] = 1.0,
+    input_scale: Annotated[float, Is[lambda x: x == 1.0]] = 1.0,
 ) -> TensorType:
-    raise NotImplementedError()
+    # del scale
+    # del input_scale
+    return op.Elu(self, alpha=alpha)
+
+
+def aten_elu__int(
+    self: IntType,
```

Comment:
@fatcat-z @xiaowuhu thoughts on reconciling data type that is not supported by the onnx native op?

Reply:
Even these are aten ops, I still suggest the function signature is ONNX compatible only. We don't need to support any Torch data type which is unsupported by ONNX in these function signatures.
We need to transfer those data types to ONNX compatible ones (Numpy/Python) before we call these aten_op() functions. If necessary, we might provide a helper lib which helps to convert data from PyTorch data format to ONNX data format. 
Such lib could be a new Python package or lives in ONNX Script package.

Reply:
Sounds like a good idea to me

Reply:
@titaiwangms is offering to help think about this aspect.